components/BottomNavigation/tests/unit/MDCBottomNavigationBarControllerTests.m

```diff
@@ -313,7 +368,56 @@ - (void)testNonSelectedViewControllersHaveViewsLazilyLoaded {
   XCTAssertFalse(childViewController2.viewLoaded);
 }
 
-#pragma mark - MDCBottomNavigationBarControllerDelegate Methods
+- (void)testEncodesChildViewControllersWhenPreservingRestorationState {
+  // Given
+  UIViewController *childViewController1 = [[UIViewController alloc] init];
+  childViewController1.restorationIdentifier = @"vc1";
```

Comment:
Can you have each VC retain some unique properties? Perhaps something simple like `title` so it can be verified we didn't mix them up?

Reply:
Sorry not following.  This tests that the archive simply contains the encoded view controller for a restoration identifier.  The mapping of restoration identifier to view controller is normally handled by UIKit.  I think it might be useful to assert that a property remains constant when we decode the view controllers in the other test?

Reply:
Ah, ok. That makes more sense. I probably misinterpreted the name of the test to indicate that it correctly encodes the child VCs, not that the coder contains the child VCs.